game-core/src/main/java/games/strategy/triplea/odds/calculator/OddsCalculatorPanel.java

```diff
@@ -500,6 +506,17 @@
     revalidate();
   }
 
+  private @Nullable PlayerId getCurrentPlayer(final History history) {
```

Comment:
Would it be better to return an Optional rather than make it nullable here?

Reply:
Done.